README.md

```diff
@@ -20,4 +21,24 @@
    sudo make install
    ```
 
+   To build a shared library which the psabpf-ctl binary will link against:
```

Comment:
The difference is only in cmake option? So there should be one instruction how to compile and separate paragraph on cmake options.